mycore-mods/src/main/java/org/mycore/mods/csl/MCRModsItemDataProvider.java

```diff
@@ -58,46 +58,33 @@
 
 public class MCRModsItemDataProvider extends MCRItemDataProvider {
 
-    private static final Logger LOGGER = LogManager.getLogger();
-
-    private static final String NON_DROPPING_PARTICLE = "nonDroppingParticle";
-
-    private static final String DROPPING_PARTICLE = "droppingParticle";
-
     public static final String USABLE_TITLE_XPATH = "mods:titleInfo[not(@altFormat) and (not(@xlink:type)" +
         " or @xlink:type='simple')]";
-
     public static final String SHORT_TITLE_XPATH = "mods:titleInfo[not(@altFormat) and (not(@xlink:type)" +
         " or @xlink:type='simple') and @type='abbreviated']";
-
     public static final String MODS_RELATED_ITEM_XPATH = "mods:relatedItem/";
-
     public static final String MODS_ORIGIN_INFO_PUBLICATION = "mods:originInfo[@eventType='publication' or not" +
         "(@eventType)]";
-
     public static final String NONE_TYPE = "none";
-
     public static final String URN_RESOLVER_LINK = "https://nbn-resolving.org/";
-
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static final String NON_DROPPING_PARTICLE = "nonDroppingParticle";
+    private static final String DROPPING_PARTICLE = "droppingParticle";
     private static final Set<String> KNOWN_UNMAPPED_PERSON_ROLES = MCRConfiguration2
         .getString("MCR.CSL.KnownUnmappedPersonRoles")
         .stream()
         .flatMap(str -> Stream.of(str.split(",")))
         .collect(Collectors.toUnmodifiableSet());
-
-    private MCRMODSWrapper wrapper;
-
-    private String id;
-
     private final Set<String> nonDroppingParticles = MCRConfiguration2.getString("MCR.CSL.NonDroppingParticles")
         .stream()
         .flatMap(str -> Stream.of(str.split(",")))
         .collect(Collectors.toSet());
-
     private final Set<String> droppingParticles = MCRConfiguration2.getString("MCR.CSL.DroppingParticles")
         .stream()
         .flatMap(str -> Stream.of(str.split(",")))
         .collect(Collectors.toSet());
+    private MCRMODSWrapper wrapper;
```

Comment:
Was this code formatted? - I would expect an empty line between the variable declarations

Reply:
The code was formated. I just checked again. Formatting it again does not add empty lines either.